dali/pipeline/operator/op_schema.cc

```diff
@@ -76,6 +97,10 @@ OpSchema::OpSchema(DefaultSchemaTag) : name_(""), default_(true) {
   AddInternalArg("default_cuda_stream_priority", "Default cuda stream priority", 0);  // deprecated
   AddInternalArg("checkpointing", "Setting to `true` enables checkpointing", false);
 
+  AddInternalArg("preserve_name", R"(When true, the operator cannot be renamed.
+This disables merging this operator with another one with a different name.)",
+                 false);
```

Comment:
Shouldn't this be the case **always** when an operator was explicitly named by the user?

I understand that having two equal ops (Readers?) but with different names is not a clever way to use DALI, but it is possible for the user to do that. Let's say that for whatever reason I have two readers doing the same work, but named differently ("Reader1", "Reader2"). I wrapped the pipeline into something (an iterator?) that relies on a presence of "Reader2" in the pipeline. This is not a good code, but it works. And after CSE it doesn't

Reply:
That's exactly how it works. If you specify a name manually, then `preserve_name` is set to `True`.

Reply:
Ah, I see:
```
self._spec.AddArg("preserve_name", not self._autoname)
```
Thank you!